content/solids/components/voxel-painter.ts

```diff
@@ -421,6 +453,26 @@ export class VoxelPainter extends CustomElementView {
 
                 this.updateApplet();
               }
+
+            }).promise.catch((reason) => console.log('anim promise error:', reason)).then(() => {
```

Comment:
Remove console log. Why do we need to change the face colours during animation end?